### DISCUSSION TITLE
After T.ToUndirected, the data.is_undirected() is still False

### DISCUSSION THREAD

**Discussion by PolarisRisingWar** [pyg-team/pytorch_geometric#5174] [Q&A]
This is my code (in which data is a HeteroData object):
```python
print(data.is_undirected())
data=T.ToUndirected()(data)
print(data.is_undirected())
```
And the output is:
```
False
False
```
I think there must have some problems...

**Comment by rusty1s** [comment]
I think this is already fixed on lastest master/nightly. Can you try to install latest PyG to see if that is indeed the case? Let me know!

**Comment by PolarisRisingWar** [comment]
This is my action list:
1. I've used conda install, so at first I used 'conda uninstall pyg`
2. My torch is 1.11.0, cudatoolkit is 10.2, so `pip install torch-scatter -f https://data.pyg.org/whl/torch-1.11.0+cu102.html` `pip install torch-sparse -f https://data.pyg.org/whl/torch-1.11.0+cu102.html` `pip install pyg-nightly`

Then I found it works! The output now is
```
False
True
```

**Reply by rusty1s** [reply]
Super. I will craft a new PyG release with this changes soon.
